dhall/src/Dhall/Hash.hs

```diff
@@ -27,7 +28,7 @@ hash censor = do
 
     resolvedExpression <- State.evalStateT (Dhall.Import.loadWith parsedExpression) status
 
-    case Dhall.TypeCheck.typeOf resolvedExpression of
+    case Dhall.TypeCheck.typeWith Context.empty resolvedExpression of
```

Comment:
I don't mind, but what's the motivation for this change?

Reply:
It's a mistake which I'll undo.  I made this change as part of a refactor to add a new argument to `typeWith`, but then realized that I could restructure this change to not require argument passing.  However, I forgot to undo this change along the way